app/models/provider.rb

```diff
@@ -60,7 +60,7 @@ class Provider < ApplicationRecord
           -> { published.latest_published_at },
           class_name: "ProviderEnrichment",
           inverse_of: 'provider'
-  has_many :courses
+  has_many :courses, -> { kept }
```

Comment:
after discussion - we'll need to clear this scope for apiv1

Reply:
Do we send UCAS courses that haven't been published yet? If we do – why?